en/references/tested-judge/template-description/README.md

```diff
@@ -0,0 +1,392 @@
+---
+title: "TESTed description templates"
+description: "TESTed description templates"
+---
+
+# Documentation description templates
+Description templates are used to only write a description once.
+That description can be instantiated foreach programming language that is necessary with the correct names and style
+conventions.
+
+The description templates will be written using the Mako templating system.
+TESTed supported two markup languages for the exercises: Markdown and HTML.
+The descriptions could be both written in English and Dutch.
+
+Which parts of the description should be generated for a programming language?
+
+- **Function names**:
+  The style conventions for function names are programming language depended.
+  Example: Python uses the `snake_case` convention, Java uses the `camelCase` convention.
+- **Type names**:
+  One of the larger difference between programming languages are the names that are used for the same datatype.
+  Example 8-bit integers: Java uses the name `byte`, Haskell uses the name `Data.Int.Int8`.
+- **Natural type names**:
```

Comment:
Is this the term for these types? Perhaps something like "Type names in natural language"